source/features/open-issue-to-latest-comment.tsx

```diff
@@ -9,6 +11,13 @@ function init(): void {
 	}
 }
 
+function initDashboard(): void {
+	for (const icon of select.all('.js-recent-activity-container :not(a) > div > .octicon-comment')) {
+		const url = icon.closest('li')!.querySelector('a')!.pathname + '#partial-timeline';
+		wrap(icon.parentElement!, <a className="muted-link rgh-latest-comment" href={url}/>);
```

Comment:
Does this work?

```suggestion
		wrapAll(icon.parentElement!.childNodes, <a className="muted-link rgh-latest-comment" href={url}/>);
```

The parentElement has a `text-gray` class that overrides our `muted-link`. It's best to have the `a` right around the icon and text node only.

Reply:
If that doesn't work, something like:
```suggestion
		const container = icon.parentElement!;
		const link = <a className="muted-link rgh-latest-comment" href={url}/>;
		icon.parentElement!.append(link);
		link.append(icon, icon.nextSibling);
```

Reply:
I didnt wrap (pun intended) exactly what you said. I will test it now